sw/simulation/agents/gas_agent/README.md

```diff
@@ -0,0 +1,2 @@
+#gas_agent
+Please include a description of the agent here, both for yourself and for future users!
```

Comment:
Write the readme please.

Also, I see that you have made your own agent after all. So please revert back the changes on particle and particle_oriented for the pull request.